ld/nodemcu.ld

```diff
@@ -103,7 +103,11 @@ SECTIONS
     *(.init.literal)
     *(.init)
     
-    *sdk/esp_iot_sdk_*lib/lib*.a:*(.literal .text)
+    *libmain.a:*(.literal .text)
+    *libnet80211.a:*(.literal .text)
+    *libphy.a:*(.literal .text)
+    *libpp.a:*(.literal .text)
+    *libgcc.a:*(.literal .text)
```

Comment:
@jmattsson Johny, as I reviewed, a lot of Espressif code is a mess. I think there was some internal Espressif's issues e.g. "radius of the developers hands curvature does not comply with the platform requirements" :)
Anyway for WPS we decide that it works when fully putted in irom. And for most places it right (with exceptions for interrupts-related stuff). I haven't see any of interrupt-related code here.